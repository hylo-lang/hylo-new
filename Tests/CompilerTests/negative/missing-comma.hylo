//! no-std

fun f<T U>(_ x: T, _ y: U) {}
