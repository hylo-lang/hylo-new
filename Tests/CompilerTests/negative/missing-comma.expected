negative/missing-comma.hylo:3.8: error: expected ','
fun f<T U>(_ x: T, _ y: U) {}
       ^

